forge-gui/res/cardsfolder/upcoming/ogre_battlecaster.txt

```diff
@@ -0,0 +1,16 @@
+Name:Ogre Battlecaster
+ManaCost:2 R
+Types:Creature Ogre Shaman
+PT:3/3
+K:First Strike
+T:Mode$ Attacks | ValidCard$ Card.Self | Execute$ TrigPlay | TriggerDescription$ Whenever CARDNAME attacks, you may cast target instant or sorcery card from your graveyard by paying {R}{R} in addition to its other costs. If that spell would be put into a graveyard, exile it instead.
+SVar:TrigPlay:Mode$ Continuous | Affected$ Instant.YouCtrl,Sorcery.YouCtrl | MayPlay$ True | MayPlayAltManaCost$ R R RegularCost | EffectZone$ Battlefield | AffectedZone$ Graveyard | RememberObjects$ Targeted | SubAbility$ DTCast
```

Comment:
This needs to be DB$ Play since you have to play it now. 
Mode$ Continuous with mayplay  is for situations where you can cast that card during a defined duration for instance this turn.

Reply:
Ir's not AB$ because it has a cost?

Reply:
its a little bit different, the ability itself doesn't have a cost, but the Spell you cast has an additional Cost

Reply:
Done, please do the testing

Reply:
I can't test it until it has been merged with the engine tweaks.

Reply:
Probably best in the future not to claim cards that require engine changes if you aren't able to test

Reply:
I didn't realize it would require engine changes until after I claimed it, but yeah, I see your point.

Reply:
I've gave the card a test and it seemed to work as intended
![image](https://user-images.githubusercontent.com/67333662/207955405-8fdabdac-2a7f-4738-a197-79fccc5be3a9.png)

![image](https://user-images.githubusercontent.com/67333662/207955458-172847ef-33d9-4382-98b1-1bb7011bedd0.png)
![image](https://user-images.githubusercontent.com/67333662/207955517-32a0e1bc-0b09-4069-973b-122970013042.png)
![image](https://user-images.githubusercontent.com/67333662/207955571-fadce452-d97c-4d8e-92d7-1f2dc06abaa4.png)